packages/webapp/src/components/Table/v2/index.jsx

```diff
@@ -0,0 +1,282 @@
+                        {...columnProps}
+                      >
+                        {format ? format(row) : data}
+                      </TableCell>
+                    );
+                  })}
+                </TableRow>
+              );
+            })}
+            {emptyRows > 0 && (
+              <TableRow style={{ height: (dense ? 40 : 56) * emptyRows }}>
+                <TableCell colSpan={fullColSpan} className={styles.tableCell} />
+              </TableRow>
+            )}
+            {!showPagination && (
+              <More
+                onClickMore={onClickMore}
+                onClickLoadMore={onClickLoadMore}
+                invisibleRowCount={!showPagination && data.length - rowsPerPage}
+                dense={dense}
+                colSpan={fullColSpan}
+              />
+            )}
+            {columns.some((column) => column.id && column.Footer) && (
+              <TableRow className={styles.footer}>
+                {columns.map(({ id, align, columnProps, Footer }) => {
+                  if (!id) {
+                    return null;
+                  }
+                  return (
+                    <TableCell
+                      key={id}
+                      align={align || 'left'}
+                      className={clsx(styles.tableCell, dense && styles.dense)}
+                      {...columnProps}
+                    >
+                      {Footer}
+                    </TableCell>
+                  );
+                })}
+              </TableRow>
+            )}
+            {FooterCell ? (
+              <TableRow>
+                <TableCell className={styles.footerCell} colSpan={fullColSpan}>
+                  <FooterCell />
+                </TableCell>
+              </TableRow>
+            ) : null}
+          </TableBody>
+        </Table>
+      </TableContainer>
+      {showPagination && (
+        <TablePagination
+          rowsPerPageOptions={pageSizeOptions}
+          component="div"
+          count={data.length}
+          rowsPerPage={rowsPerPage}
+          page={page}
+          onPageChange={handleChangePage}
+          onRowsPerPageChange={handleChangeRowsPerPage}
+        />
+      )}
+    </Box>
+  );
+}
+
+EnhancedTable.propTypes = {
+  columns: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      format: PropTypes.func,
+      accessor: PropTypes.func,
+      align: PropTypes.oneOf(['left', 'right']),
+      Footer: PropTypes.node,
+      columnProps: PropTypes.object,
+    }),
+  ),
+  data: PropTypes.array,
+  showPagination: PropTypes.bool,
+  pageSizeOptions: PropTypes.arrayOf(PropTypes.number),
+  minRows: PropTypes.number,
+  dense: PropTypes.bool,
+  FooterCell: PropTypes.elementType,
+  onClickMore: PropTypes.func,
+  itemsToAddPerLoadMoreClick: PropTypes.number,
+  onRowClick: PropTypes.func,
+  /** should be true when setting column width */
+  shouldFixTableLayout: PropTypes.bool,
+  defaultOrderBy: PropTypes.string,
```

Comment:
Like this

```js
EnhancedTable.defaultProps = {
    minRows: 10,
    pageSizeOptions: [5, 10, 20, 50],
    ...
}
```

and I'd also suggest adding `isRequired` to propTypes that are not optional, like

```js
EnhancedTable.propTypes = {
    data: PropTypes.array.isRequired,
    minRows: PropTypes.array.isRequired, // This is required for the component but since it has a default in won't fail if not passed
    ...
}
```

Reply:
Ohh I thought you were saying .enum('desc', 'asc') sort of thing.

Reply:
Oh, I didn't know that!
If we do it, the default values of `minRows` and `pageSizeOptions` below are not necessary, correct?
```
const EnhancedTable({ minRows = 10, pageSizeOptions = [5, 10, 20, 50] }) {
...
```

Reply:
It seems Storybook doesn't infer defaults from defaultProps anymore 🤷‍♀️